package.json

```diff
@@ -36,6 +36,7 @@
     "test:render": "jest test/render.test.js --forceExit",
     "test:unit": "jest --testPathPattern src",
     "test:xmlserver": "http-server -c-1 -p 8085 ./examples",
+    "test:validate-xml": "find examples/ -name '*.xml' | xargs xmllint --schema schema/hyperview.xsd --noout",
```

Comment:
is this too early to chain this  to the test command?

Reply:
yes, like I mentioned some of the examples don't pass yet due to a conflict in the spec.